### ISSUE TITLE
URLs for individual models

### ISSUE THREAD

**Issue by fedorov** [modelhub-ai/modelhub-app#23]
Right now, the URL does not change as the individual models are selected. This makes it rather cumbersome to share or reference a specific model, which has repercussions on incentivizing contributions, tracking versions of the model as the user experiments with them, etc. I think it would be nice if every model had a (versioned?) URL.

@ahmedhosny I just thought I add this for the future, I know you are busy.

**Comment by ahmedhosny** [comment]
Agreed would be very nice to have this implemented. Should be straight forward through react router.
